modules/installation-azure-confidential-vms.adoc

```diff
@@ -40,33 +40,27 @@ Confidential VMs are currently not supported on 64-bit ARM architectures.
 
 .Procedure
 
-* Use a text editor to edit the `install-config.yaml` file prior to deploying your cluster and add the following stanza:
+* Edit the `install-config.yaml` file before deploying your cluster:
+
+** To enable confidential VMs only on control plane, add the following stanza:
```

Comment:
```suggestion
** Enable confidential VMs only on the control plane by adding the following stanza:
```

Reply:
I prefer the direct syntax because I am thinking of someone searching the web for "enabling confidential VMs on..."